packages/app/src/app/pages/Dashboard/Components/Repository/ImportRepository.tsx

```diff
@@ -38,15 +36,10 @@ export const ImportRepositoryCard: React.FC<{ disabled?: boolean }> = ({
       })}
       disabled={disabled}
     >
-      <Stack
-        direction="vertical"
-        align="center"
-        gap={4}
-        css={disabled ? { opacity: 0.4 } : undefined}
-      >
-        <Icon name="plus" size={32} />
-        <Text>Import repository</Text>
-      </Stack>
-    </Element>
+      <Icon color="#999999" name="plus" size={20} />
+      <Text color={disabled ? '#808080' : '#999'} size={13} weight={500}>
```

Comment:
To fix the type error:
```suggestion
      <Text color={disabled ? '#808080' : '#999'} size={13} weight="500">
```